flang/include/flang/Parser/preprocessor.h

```diff
@@ -121,6 +121,8 @@ class Preprocessor {
   std::list<std::string> names_;
   std::unordered_map<CharBlock, Definition> definitions_;
   std::stack<CanDeadElseAppear> ifStack_;
+
+  unsigned CounterValue = 0;
```

Comment:
We typically use `unsigned int` in this code. Also, the variable names should start with lower case and member variables should end with `_`.  Also, this code uses `{}` initialization.
```suggestion
  unsigned int counter_val_{0};
```